srv/components/provisioner/scripts/cortx_prep.sh

```diff
@@ -235,9 +235,10 @@ install_cortx_pkgs()
         "cortx-s3server"
         "cortx-sspl"
         "cortx-sspl-cli"
-        "python36-cortx-prvsnr"
```

Comment:
Why do we remove this package:
`python36-cortx-prvsnr`

Reply:
Also can we change the prompt as:
`nodecli>`

Reply:
>Why do we remove this package:
python36-cortx-prvsnr

This was a bug we were 2 entries for python36-cortx-prvsnr package..
so thatswhy i removed it...
it was not creating any issue though.

Reply:
>Also can we change the prompt as:
nodecli>

Here the package name is node_cli but the shell would be 'nodecli'
I have shared a sample below how it would be looking...

[root@ssc-vm-rhev4-0333 ~]# node_cli
Node cli interface
nodecli>> exit